s3/src/main/scala/geotrellis/spark/io/s3/package.scala

```diff
@@ -7,5 +7,14 @@ package object s3 {
   def makePath(chunks: String*) =
     chunks.filter(_.nonEmpty).mkString("/")
 
-  implicit class withSaveToS3Methods[K](rdd: RDD[(K, Array[Byte])]) extends SaveToS3Methods[K](rdd)
+  implicit class withSaveToS3Methods[K](rdd: RDD[(K, Array[Byte])]) {
```

Comment:
Why move this into the package? The pattern has been to have the object, abstract class, and the Implicits with the `with` prefix. Which lets you reference a `? => SaveToS3Methods` if you need it.